services/QuillLMS/config/environments/production.rb

```diff
@@ -66,13 +66,8 @@
 
   # Precompile additional assets.
   # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-  config.assets.precompile += %w(
-                                 cms.js
-                                 scorebook/*
-                                 scorebook/**/*
-                                 sign_up_email.css
-                                 admin_usage_snapshot_report_pdf.scss
-                                 )
+  # config.assets.precompile += []
```

Comment:
This comment may be helpful to people hunting for the `precompile` directive.

Reply:
Great idea!